From 73fe83c14cb257e156e5b32d703ca5734059433a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:25 -0500
Subject: [redhat] kabi: Adding symbol pci_iomap (lib/pci_iomap.c)

Message-id: <4678d0ba032674f9582b51776cc2dd52ffcd02dc.1418903403.git.poros@redhat.com>
Patchwork-id: 102265
O-Subject: [PATCH RHEL7.1 33/48] kabi: Adding symbol pci_iomap (lib/pci_iomap.c)
Bugzilla: 1160762
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160762
    https://bugzilla.redhat.com/show_bug.cgi?id=1160762

    Description:
    ============
    setup bar0 mmap when adding new instance of PCIE-RT card.
    Used in pcie_rt.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/pci_iomap
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/pci_iomap
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/pci_iomap
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pci_iomap

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/pci_iomap b/redhat/kabi/kabi-module/kabi_ppc64/pci_iomap
new file mode 100644
index 0000000..6294bd4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/pci_iomap
@@ -0,0 +1,2 @@
+#1-
+0x1a0a5fe6 pci_iomap vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/pci_iomap b/redhat/kabi/kabi-module/kabi_ppc64le/pci_iomap
new file mode 100644
index 0000000..06e90e2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/pci_iomap
@@ -0,0 +1,2 @@
+#1-
+0xf15997d8 pci_iomap vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/pci_iomap b/redhat/kabi/kabi-module/kabi_s390x/pci_iomap
new file mode 100644
index 0000000..e91dad6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/pci_iomap
@@ -0,0 +1,2 @@
+#1-
+0x99349287 pci_iomap vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pci_iomap b/redhat/kabi/kabi-module/kabi_x86_64/pci_iomap
new file mode 100644
index 0000000..5a224d7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pci_iomap
@@ -0,0 +1,2 @@
+#1-
+0x8055d058 pci_iomap vmlinux EXPORT_SYMBOL
-- 
1.7.1